### ISSUE TITLE
UI Simplification: Allow selecting Layer from the right-click menu

### ISSUE THREAD

**Issue by miina** [swissspidy/web-stories-wp#10107]
## Feature Description
See [Figma](https://www.figma.com/file/dc9xrVlRhQq4KTE9WQzU1R/_Editor---Sprint-Feature-Level-Designs_?node-id=11122%3A448445)

- [ ] Add a new menu item to the right-click menu for selecting a layer
- [ ] It displays all the Layers that are under the cursor at the time of the click
- [ ] Make sure it's possible to use both menus with keyboard (needs reworking the current navigation logic)

NOTE: We will not add the icon to the Layers currently and will only use text (and the tick for the selected layer). See [Slack message](https://xwp.slack.com/archives/C01ESBB8ZN1/p1642088410023000?thread_ts=1641470719.003600&cid=C01ESBB8ZN1).

## Additional Context
<img width="404" alt="Screenshot 2022-01-06 at 12 07 46" src="https://user-images.githubusercontent.com/3294597/148366274-efb317dd-f8c5-43af-9517-45f5eca58155.png">

**Comment by andrew-askedall** [comment]
No, it should only display the layers under the cursor at time of click.

**Comment by miina** [comment]
> No, it should only display the layers under the cursor at time of click.

@andrew-askedall It's also possible to open the right-click menu via a shortcut. In this case, the menu is opened from the top-left corner of the currently selected element. Should we display the layers under that corner in this case? Since there's no click then it won't be possible to take the layers based on the cursor.

Edit:
One more question -- about keyboard interaction. Currently, when using arrow keys (including left-right), the focus will move between the items of the same menu. Let me know how should we change this when a sub-menu is open.
E.g. should we allow moving into / out of the submenu using Tab / Shift + Tab? Or should we fully change the current logic and allow moving to the submenu using left-right arrow keys as well (this would mean removing using left-right arrow keys for navigation when there is no submenu as well)?

Thoughts?

**Comment by andrew-askedall** [comment]
I think the left or right arrow keys make sense to dig into a submenu. I would do whatever makes it the most accessible and is best practice. I think displaying layers under that top corner makes sense.

**Comment by felipebochehin87** [comment]
Tested against https://stories-qa-wordpress-amp.pantheonsite.io/wp-admin/, using PR #10199.

1. Added elements to the canvas
2. Added an image on top of the other elements
3. Right-clicked on the image - Select Layer is displayed
4. Clicked on Select Layer - I was able to select the elements using the options in the menu
5. Right-click menu from the Layer panel didn't not display the Select Layer item in the menu - as expected
6. Layers that are not under the cursor while opening the right-click menu were not displayed in the submenu - as expected
7. Accessibility working as expected when using the arrow keys
